lib/src/utils/file_utils.dart

```diff
@@ -21,19 +21,19 @@ File getPubspecFile() {
 }
 
 /// Gets arb file for the given locale.
-File getArbFileForLocale(String locale) {
+File getArbFileForLocale(String locale, String arbDir) {
   var rootDirPath = getRootDirectoryPath();
-  var arbFilePath = path.join(rootDirPath, 'lib', 'l10n', 'intl_$locale.arb');
-  var arbFile = File(arbFilePath);
+  var arbFileDir = path.join(rootDirPath, arbDir, 'intl_$locale.arb');
+  var arbFile = File(arbFileDir);
 
   return arbFile.existsSync() ? arbFile : null;
 }
 
 /// Creates arb file for the given locale.
-Future<File> createArbFileForLocale(String locale) async {
+Future<File> createArbFileForLocale(String locale, String arbPath) async {
   var rootDirPath = getRootDirectoryPath();
-  var arbFilePath = path.join(rootDirPath, 'lib', 'l10n', 'intl_$locale.arb');
-  var arbFile = File(arbFilePath);
+  var arbFileDir = path.join(rootDirPath, arbPath, 'intl_$locale.arb');
```

Comment:
I don't want to be a show stopper, but why we changed the variable name here from `arbFilePath` to `arbFileDir`?
I'm aware that everything will work, but it may be misleading for future changes...

Reply:
I guess after so many Path -> Dir, I got confused 😅